src/Content/Backend/Solution/Monaco.Template.Backend.sln

```diff
@@ -64,6 +64,8 @@ EndProject
 #if (filesSupport)
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Monaco.Template.Backend.Common.BlobStorage", "Monaco.Template.Backend.Common.BlobStorage\Monaco.Template.Backend.Common.BlobStorage.csproj", "{42E51D47-B82F-4A92-B1E5-CD8BE44DE6F0}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Monaco.Template.Backend.Common.BlobStorage.Tests", "Monaco.Template.Backend.Common.BlobStorage.Tests\Monaco.Template.Backend.Common.BlobStorage.Tests.csproj", "{D8623B90-59C1-4753-A0E6-F2DBD4305C9B}"
```

Comment:
Please add the GUID `D8623B90-59C1-4753-A0E6-F2DBD4305C9B` to the list of GUIDs on `template.json` to make sure this gets randomized on a new solution.